torchvision/models/detection/rpn.py

```diff
@@ -164,13 +162,8 @@ def forward(self, image_list, feature_maps):
                     torch.tensor(image_size[1] // g[1], dtype=torch.int64, device=device)] for g in grid_sizes]
         self.set_cell_anchors(dtype, device)
         anchors_over_all_feature_maps = self.cached_grid_anchors(grid_sizes, strides)
-        anchors = torch.jit.annotate(List[List[torch.Tensor]], [])
-        for i, (image_height, image_width) in enumerate(image_list.image_sizes):
-            anchors_in_image = []
-            for anchors_per_feature_map in anchors_over_all_feature_maps:
-                anchors_in_image.append(anchors_per_feature_map)
-            anchors.append(anchors_in_image)
-        anchors = [torch.cat(anchors_per_image) for anchors_per_image in anchors]
+        anchors: List[torch.Tensor] = [torch.cat(anchors_over_all_feature_maps)
+                                       for _ in range(len(image_list.image_sizes))]
         # Clear the cache in case that memory leaks.
```

Comment:
Here I found that variable `anchors_in_image` is the copy of variable `anchors_over_all_feature_maps`, so I tried to simplify the code.